### PR TITLE
switched max-widths to vars

### LINKED ISSUE #292: Connect all media queries to CSS variable breakpoints
# Feature request
**Is your feature request related to a problem? Please describe.**
There exist a large slew of hard-coded viewport checks within media queries around our stylesheets. Hard-coding isn't reuse. It's difficult to refactor, and prone to developer mistakes.

**Describe the solution you'd like**
Now that we have breakpoints defined via `common/styles/variables`, I'd like them leveraged so that we have a manageable system of media queries. 

**Example:**
```globalStyles.css```
```css
/* @media screen and (max-width: 480px) { */
@media screen and (max-width: var(--smallViewportWidth)) {
  html {
    .font-size: 14px !important;
  }
}

/* @media screen and (min-width: 481px) and (max-width: 960px) { */
@media screen and (min-width: var(--smallViewportWidth)) and (max-width: var(--largeViewportWidth)) {
  html {
    font-size: 16px !important;
  }
}
```

**Describe alternatives you've considered**

**Additional context**
You may notice visual regression as you go. This is okay. We need to make a certain amount of changes before we attempt to tackle the various style issues on the website. Just change the code, and document this issues in your PR please 🙏

### PR DESCRIPTION
**Pull request by markchernov**
# Description of changes

Changes max-width px media queries to vars
# Issue Resolved

Fixes #292

### CODE REVIEW

**Comment by kylemh** [comment]
🤖 **This is a bot** 🤖<br /><br />🎉 Deployed Storybook preview! 🎉<br /><br />Click the link at the bottom of this comment to see it live.

Built with commit c62030b6a6beb7a7a839df496950234b5e19e742

https://deploy-preview-334--operation-code-storybook.netlify.com

File: components/Cards/ImageCard/ImageCard.css [hunk 1]
```diff
@@ -1,3 +1,5 @@
+@import "../../../common/styles/media-queries.css";
```
[the remark this answers is not in the document]
Comment (markchernov) [reply components/Cards/ImageCard/ImageCard.css]: @import "common/styles/media-queries.css";
@import "/common/styles/media-queries.css";
None of these seem to work

[the remark this answers is not in the document]
Comment (markchernov) [reply components/Cards/ImageCard/ImageCard.css]: I'll keep researching. So far no luck with the above options

**Comment by kylemh** [comment]
I'll make  a separate issue for removing the imports.

File: components/Cards/ImageCard/ImageCard.css (hunk 1 above)
[the remark this answers is not in the document]
Comment (kylemh) [reply components/Cards/ImageCard/ImageCard.css]: Could be related to #316

**Comment by markchernov** [comment]
Sounds good, hope new css-loader will help

**Pull request merged**